hrabsence/hrabsence.php

```diff
@@ -235,6 +235,8 @@ function hrabsence_civicrm_install() {
   );
   civicrm_api3('message_template', 'create', $msg_params);
 
+  _hrabsence_add_scheduled_jobs();
```

Comment:
In this case, I'm going with the option which will take less time: don't change anything :D